cms/static/js/factories/program_editor.js

```diff
@@ -0,0 +1,9 @@
+define([
+    'js/programs/program_admin_app'
+],
+function(program_admin_app) {
+  'use strict';
+  return function(){
+    new program_admin_app();
```

Comment:
@andy-armstrong I think you are correct. Once I have the sandbox spined up, I'll test just use `program_admin_app` as the factory